src/hashesorg.py

```diff
@@ -0,0 +1,26 @@
+def HashesOrg(hashed, key):
+    try:
+        output = requests.get(f"https://hashes.org/api.php?key={key}&query={hashed}", timeout=5).text
+    except:
+        print("Couldn't connect")
+        return False
+
+    # Check for false positive
+
+    if "null" in output:
+        print("Hash not found") ; return False
+    if "" == output:
+        print("Hash seems to be a plain?") ; return False
+
+    final_output = output.split('":"')[2].split('","')[0]
+
+    # Check for Hex
+
+    if "$HEX[" in final_output:
+        final_output = bytearray.fromhex(final_output[5 : len(final_output) - 1]).decode()
+
+    print(f"Found - {final_output}")
+
+    return True

```

Comment:
Don't return True, don't print the output -- return the output :)

Reply:
1. Loop over decryptors
2. Multi-thread `class.decrypt()`
3. Store all results in a list of outputs.
4. Find the result which returned True and is the plaintext.